### DISCUSSION TITLE
Segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by Ali141272** [neperfepx/neper#656] [Q&A] [answered]
Hello @rquey and @smallrain1996,
How to resolved this issue, because when I change the values of mean and standard deviation then I have face this problem in every steps?
My Gmsh version is 4.9.5 and Neper version is 4.4.1.
I use the following commands:

neper -T -n 1260 -morpho gg -reg 1
neper -T -n 1260 -morpho "diameq:lognormal(0.1,0.06),1-sphericity:lognormal(0.130,0.03)" -domain "square(0.05,0.1)" -dim 2 -o 222

Thank you.
![1](https://user-images.githubusercontent.com/102291665/224032529-c565c5df-28ff-4f23-b46c-7b200e7fd1f4.png)

**Comment by rquey** [comment, marked as answer]
It works with Neper version 4.5.0. Please update.

**Reply by Ali141272** [reply]
Thank you Professor for your positive response.
I have downloaded the new version 4.5.0 how to upgrade?

**Reply by rquey** [reply]
There is no "upgrade" *per se*. Install the new version just as you installed the previous one, and it should be overwritten. Then check that you have the correct version by running `neper --version`.

**Reply by Ali141272** [reply]
Ok Professor Thanks once again

**Reply by Ali141272** [reply]
Hello Professor Dr. @rquey ,
If I upgrade my Neper version from 4.4.1.to 4.5.0, should I delete my old version of Neper from my repositories or no need to delete?
For Neper upgradation I use the following commands, but I have bit confusion whether first I use the installation dependencies or after the installation of Neper on Ubuntu?
```
#Install dependancies 
$ sudo apt-get install libgsl-dev
$ sudo apt-get install libnlopt-dev
$ sudo apt-get install libomp-dev
$ sudo apt-get install libscotch-dev
$ sudo apt-get install libpthread-stubs0-dev
```
#Install Neper
---Download and extract Neper source code from---
	https://neper.info/downloads.html
```
$ cd neper-4.5.0/src
$ mkdir build
$ cd build
$ cmake ..
$ sudo make
$ sudo make install
```

Thank you.

**Reply by rquey** [reply]
To upgrade Neper, you don't have to re-install the dependencies. When you install Neper, `sudo make install` will overwrite the existing Neper binary with the new version.

**Reply by Ali141272** [reply]
Professor, Thanks once again for your positive response.
